packages/service/lib/stackable.js

```diff
@@ -54,27 +64,13 @@ class ServiceStackable {
     return this.app
   }
 
-  async getOpenapiSchema () {
-    await this.init()
-    await this.app.ready()
-    return this.app.swagger ? this.app.swagger() : null
-  }
-
-  async getGraphqlSchema () {
-    await this.init()
-    await this.app.ready()
-    return this.app.graphql ? printSchema(this.app.graphql.schema) : null
-  }
```

Comment:
I really think we should maintain those, but just use a global if this is not possible.